.gitignore

```diff
@@ -1 +1,3 @@
-*build
\ No newline at end of file
+*build
+*out
```

Comment:
Can you explain the motivation for this entry?

Reply:
From what I've seen, it's idiomatic at times to use `out/build` as an output directory; especially when using CMake presets. This allows you to put your installation directory inside `out` as well (for testing purposes) without polluting your system environment. 

So in some hypothetical preset:

```json
{
	"configurePresets" : [
		{
		    "name": "base",
		    "hidden": true,
		    "generator": "Ninja",
		    "binaryDir": "${sourceDir}/out/build/${presetName}",
		    "installDir": "${sourceDir}/out/install/${presetName}",
		    "cacheVariables": {
		        "BUILD_TESTING": "ON",
		        "CMAKE_CXX_STANDARD": "23",
		        "CMAKE_EXPORT_COMPILE_COMMANDS": "ON"
		    },
		    "architecture": {
		        "strategy": "external",
		        "value": "x64"
		    },
		    "toolset": {
		        "strategy": "external",
		        "value": "x64"
		    }
		}
	]
}
```